### PR TITLE
Update docs for `secretsMap` option support

### PR DESCRIPTION
**Pull request by Huanli-Meng**
### Motivation
In Function Mesh V0.1.9 release, [secrets are supported](https://github.com/streamnative/function-mesh-worker-service/pull/21). Therefore, update docs for this code PR.

### Modifications:
- Update docs for secrets supported on function mesh worker service.
  - Related release: latest
- Refine authentication descriptions 
  - Related releases: v0.1.5- latest
- Update typos, code format and heading levels:
  - Related releases: all releases

### CODE REVIEW

File: docs/connectors/io-crd-config/sink-crd-config.md [hunk 1]
```diff
@@ -63,30 +63,44 @@ If the node where a Pod is running has enough of a resource available, it is pos
...
+kind: Secret
+metadata:
+  name: <secret_name>
+type: Opaque
+```
 
-1. Create a secret or use an existing one. Multiple Pods can reference the same secret.
-2. Modify your Pod definition in each container, which you want to consume the value of a secret key, to add an environment variable for each secret key that you want to consume.
-3. Modify your image and or command line so that the program looks for values in the specified environment variables.
+This table lists configurations about the `SecretsMap` field.
```
Comment (freeznet) [inline docs/connectors/io-crd-config/sink-crd-config.md:79]: `SecretsMap` is defined as a `Map` struct with `String` keys and `SecretReference` values. The `key` will indicate the environment value in the container, and the `SecretReference` is defined as below.

| Field | Description |
 | --- | --- |
 | `path` | The name of the secret in the pod's namespace to select from. |
 | `key` | The key of the secret to select from.  Must be a valid secret key. |

For example, the user has a Kubernetes Secret named `credential-secret` defined as below:

```yaml
 apiVersion: v1
 data:
   username: foo
   password: bar
 kind: Secret
 metadata:
   name: credential-secret
 type: Opaque
 ```

To use it in Pulsar Functions in a security way, simply define the `SecretsMap` in the Custom Resource:

```
secretsMap:
  username:
    path: credential-secret
    key: username
  password:
    path: credential-secret
    key: password
```

Then in the Pulsar Functions and Connectors, you can call `context.getSecret("username")` to get the secret value.

**Review by freeznet — COMMENTED** [review]
please check my comment, I only made suggestions on sink doc and the changes can be used for source and function as well because the content of sink, source and function should be similar.

**Comment by Huanli-Meng** [comment]
@freeznet Thanks for your valuable comments. Docs have been updated based on you comments. PTAL again and feel free to approve it if there is no more comment. Thanks.

File: docs/connectors/io-crd-config/sink-crd-config.md [hunk 2]
```diff
@@ -63,30 +63,60 @@ If the node where a Pod is running has enough of a resource available, it is pos
...
+type: Opaque
+```
+
+To use it in Pulsar Functions in a security way, you can define the `SecretsMap` in the Custom Resource:
+
+```yaml
+secretsMap:
+  username:
+    path: credential-secret
+    key: foo
```
Comment (freeznet) [inline docs/connectors/io-crd-config/sink-crd-config.md:94]:
```suggestion
    key: username
```

File: docs/connectors/io-crd-config/sink-crd-config.md [hunk 3]
```diff
@@ -63,30 +63,60 @@ If the node where a Pod is running has enough of a resource available, it is pos
...
+  password:
+    path: credential-secret
+    key: bar
```
Comment (freeznet) [inline docs/connectors/io-crd-config/sink-crd-config.md:97]:
```suggestion
    key: password
```

File: docs/connectors/io-crd-config/sink-crd-config.md [hunk 4]
```diff
@@ -63,30 +63,60 @@ If the node where a Pod is running has enough of a resource available, it is pos
...
+```
+
+Then, in the Pulsar Functions and Connectors, you can call `context.getSecret("username")` to get the secret value.
```
Comment (freeznet) [inline docs/connectors/io-crd-config/sink-crd-config.md:100]:
```suggestion
Then, in the Pulsar Functions and Connectors, you can call `context.getSecret("username")` to get the secret value `"foo"`.
```

**Review by freeznet — COMMENTED** [review]
sorry, i have made some minor changes, to fix the wrong resouce ref in the custom resource.

**Comment by Huanli-Meng** [comment]
> sorry, i have made some minor changes, to fix the wrong resouce ref in the custom resource.

It's OK. I have updated the doc based on your comments. PTAL again. Thanks

**Review by freeznet — APPROVED** [review]
thanks, lgtm

File: docs/connectors/io-crd-config/sink-crd-config.md [hunk 5]
```diff
@@ -63,30 +63,60 @@ If the node where a Pod is running has enough of a resource available, it is pos
 
 ## Secrets
 
-In Function Mesh, the secret is defined through a secretsMap. To use a secret, a Pod needs to reference the secret. Pods can consume secretsMaps as environment variables in a volume. You can specify the `data` field when creating a configuration file for a secret.
+Function Mesh provides the `SecretsMap` field for Function, Source and Sink in the CRD definition. You can refer to the created secrets under the same namespace and the controller can include those referred secrets. The secrets are provide by `EnvironmentBasedSecretsProvider` which can be used by `context.getSecret()` in Pulsar functions and connectors.
```
Comment (teachwrite) [inline docs/connectors/io-crd-config/sink-crd-config.md:66]: need a comma after "Source" in the first sentence ("...Function, Source, and Sink..") and a comma before which in the last sentence "... `EnvironmentBasedSecretsProvider`, which..."

File: docs/connectors/io-crd-config/sink-crd-config.md [hunk 6]
```diff
@@ -63,30 +63,60 @@ If the node where a Pod is running has enough of a resource available, it is pos
...
+data:
+  username: foo
+  password: bar
+kind: Secret
+metadata:
+  name: credential-secret
+type: Opaque
+```
+
+To use it in Pulsar Functions in a security way, you can define the `SecretsMap` in the Custom Resource:
```
Comment (teachwrite) [inline docs/connectors/io-crd-config/sink-crd-config.md:88]: "in a secure way"

File: docs/connectors/io-crd-config/sink-crd-config.md [hunk 7]
```diff
@@ -63,30 +63,60 @@ If the node where a Pod is running has enough of a resource available, it is pos
...
+    path: credential-secret
+    key: password
+```
+
+Then, in the Pulsar Functions and Connectors, you can call `context.getSecret("username")` to get the secret value (`foo`).
+
+## Authentication
 
-Pulsar clusters support using TLS or other authentication plugin for authentication.
+Function Mesh provides the `TLSSecret` and `AuthSecret` fields for Function, Source and Sink in the CRD definition. You can configure TLS encryption and/or TLS authentication using the following configurations.
```
Comment (teachwrite) [inline docs/connectors/io-crd-config/sink-crd-config.md:104]: need a comma after "Source" in the first sentence ("...Function, Source, and Sink..")

File: docs/connectors/io-crd-config/source-crd-config.md [hunk 8]
```diff
@@ -56,30 +56,60 @@ If the node where a Pod is running has enough of a resource available, it's poss
 
 ## Secrets
 
-In Function Mesh, the secret is defined through a secretsMap. To use a secret, a Pod needs to reference the secret. Pods can consume secretsMaps as environment variables in a volume. You can specify the `data` field when creating a configuration file for a secret.
+Function Mesh provides the `SecretsMap` field for Function, Source and Sink in the CRD definition. You can refer to the created secrets under the same namespace and the controller can include those referred secrets. The secrets are provide by `EnvironmentBasedSecretsProvider` which can be used by `context.getSecret()` in Pulsar functions and connectors.
```
Comment (teachwrite) [inline docs/connectors/io-crd-config/source-crd-config.md:59]: need a comma after "Source" in the first sentence ("...Function, Source, and Sink..") and a comma before which in the last sentence "... EnvironmentBasedSecretsProvider, which..."

File: docs/connectors/io-crd-config/source-crd-config.md [hunk 9]
```diff
@@ -56,30 +56,60 @@ If the node where a Pod is running has enough of a resource available, it's poss
...
+    path: credential-secret
+    key: password
+```
+
+Then, in the Pulsar Functions and Connectors, you can call `context.getSecret("username")` to get the secret value (`foo`).
+
+## Authentication
 
-Pulsar clusters support using TLS or other authentication plugin for authentication.
+Function Mesh provides the `TLSSecret` and `AuthSecret` fields for Function, Source and Sink in the CRD definition. You can configure TLS encryption and/or TLS authentication using the following configurations.
```
Comment (teachwrite) [inline docs/connectors/io-crd-config/source-crd-config.md:97]: need a comma after "Source" in the first sentence ("...Function, Source, and Sink..")

File: docs/functions/function-crd.md [hunk 10]
```diff
@@ -86,30 +84,60 @@ If the node where a Pod is running has enough of a resource available, it's poss
 
 ## Secrets
 
-In Function Mesh, the secret is defined through a secretsMap. To use a secret, a Pod needs to reference the secret. Pods can consume secretsMaps as environment variables in a volume. You can specify the `data` field when creating a configuration file for a secret. 
+Function Mesh provides the `SecretsMap` field for Function, Source and Sink in the CRD definition. You can refer to the created secrets under the same namespace and the controller can include those referred secrets. The secrets are provide by `EnvironmentBasedSecretsProvider` which can be used by `context.getSecret()` in Pulsar functions and connectors.
```
Comment (teachwrite) [inline docs/functions/function-crd.md:87]: need a comma after "Source" in the first sentence ("...Function, Source, and Sink..") and a comma before which in the last sentence "... EnvironmentBasedSecretsProvider, which..."

File: docs/functions/function-crd.md [hunk 11]
```diff
@@ -86,30 +84,60 @@ If the node where a Pod is running has enough of a resource available, it's poss
...
+data:
+  username: foo
+  password: bar
+kind: Secret
+metadata:
+  name: credential-secret
+type: Opaque
+```
+
+To use it in Pulsar Functions in a security way, you can define the `SecretsMap` in the Custom Resource:
```
Comment (teachwrite) [inline docs/functions/function-crd.md:109]: "in a secure way"

File: versioned_docs/version-0.1.5/connectors/io-crd-config/sink-crd-config.md [hunk 12]
```diff
@@ -60,32 +60,24 @@ When you specify a function or connector, you can optionally specify how much of
...
-In Function Mesh, the secret is defined through a secretsMap. To use a secret, a Pod needs to reference the secret. Pods can consume secretsMaps as environment variables in a volume. You can specify the `data` field when creating a configuration file for a secret.
-
-To use a secret in an environment variable in a Pod, follow these steps.
-
-1. Create a secret or use an existing one. Multiple Pods can reference the same secret.
-2. Modify your Pod definition in each container, which you want to consume the value of a secret key, to add an environment variable for each secret key that you want to consume.
-3. Modify your image and or command line so that the program looks for values in the specified environment variables.
-
-Pulsar clusters support using TLS or other authentication plugin for authentication.
+Function Mesh provides the `TLSSecret` and `AuthSecret` fields for Function, Source and Sink in the CRD definition. You can configure TLS encryption and/or TLS authentication using the following configurations.
```
Comment (teachwrite) [inline versioned_docs/version-0.1.5/connectors/io-crd-config/sink-crd-config.md:65]: need a comma after "Source" in the first sentence ("...Function, Source, and Sink..")

File: versioned_docs/version-0.1.5/connectors/io-crd-config/source-crd-config.md [hunk 13]
```diff
@@ -52,32 +52,24 @@ When you specify a function or connector, you can optionally specify how much of
...
-In Function Mesh, the secret is defined through a secretsMap. To use a secret, a Pod needs to reference the secret. Pods can consume secretsMaps as environment variables in a volume. You can specify the `data` field when creating a configuration file for a secret.
-
-To use a secret in an environment variable in a Pod, follow these steps.
-
-1. Create a secret or use an existing one. Multiple Pods can reference the same secret.
-2. Modify your Pod definition in each container, which you want to consume the value of a secret key, to add an environment variable for each secret key that you want to consume.
-3. Modify your image and/or command line so that the program looks for values in the specified environment variables.
-
-Pulsar clusters support using TLS or other authentication plugin for authentication.
+Function Mesh provides the `TLSSecret` and `AuthSecret` fields for Function, Source and Sink in the CRD definition. You can configure TLS encryption and/or TLS authentication using the following configurations.
```
Comment (teachwrite) [inline versioned_docs/version-0.1.5/connectors/io-crd-config/source-crd-config.md:57]: need a comma after "Source" in the first sentence ("...Function, Source, and Sink..")

File: versioned_docs/version-0.1.5/functions/function-crd.md [hunk 14]
```diff
@@ -83,32 +81,24 @@ When you specify a function or connector, you can optionally specify how much of
...
-
-To use a secret in an environment variable in a Pod, follow these steps.
-
-1. Create a secret or use an existing one. Multiple Pods can reference the same secret.
-2. Modify your Pod definition in each container, which you want to consume the value of a secret key, to add an environment variable for each secret key that you want to consume.
-3. Modify your image and/or command line so that the program looks for values in the specified environment variables.
+## Authentication
 
-Pulsar clusters supports using TLS or other authentication plugin for authentication.
+Function Mesh provides the `TLSSecret` and `AuthSecret` fields for Function, Source and Sink in the CRD definition. You can configure TLS encryption and/or TLS authentication using the following configurations.
```
Comment (teachwrite) [inline versioned_docs/version-0.1.5/functions/function-crd.md:86]: need a comma after "Source" in the first sentence ("...Function, Source, and Sink..")

File: versioned_docs/version-0.1.6/connectors/io-crd-config/sink-crd-config.md [hunk 15]
```diff
@@ -60,32 +60,24 @@ When you specify a function or connector, you can optionally specify how much of
...
-In Function Mesh, the secret is defined through a secretsMap. To use a secret, a Pod needs to reference the secret. Pods can consume secretsMaps as environment variables in a volume. You can specify the `data` field when creating a configuration file for a secret.
-
-To use a secret in an environment variable in a Pod, follow these steps.
-
-1. Create a secret or use an existing one. Multiple Pods can reference the same secret.
-2. Modify your Pod definition in each container, which you want to consume the value of a secret key, to add an environment variable for each secret key that you want to consume.
-3. Modify your image and or command line so that the program looks for values in the specified environment variables.
-
-Pulsar clusters support using TLS or other authentication plugin for authentication.
+Function Mesh provides the `TLSSecret` and `AuthSecret` fields for Function, Source and Sink in the CRD definition. You can configure TLS encryption and/or TLS authentication using the following configurations.
```
Comment (teachwrite) [inline versioned_docs/version-0.1.6/connectors/io-crd-config/sink-crd-config.md:65]: need a comma after "Source" in the first sentence ("...Function, Source, and Sink..")

File: versioned_docs/version-0.1.6/connectors/io-crd-config/source-crd-config.md [hunk 16]
```diff
@@ -52,32 +52,24 @@ When you specify a function or connector, you can optionally specify how much of
...
-In Function Mesh, the secret is defined through a secretsMap. To use a secret, a Pod needs to reference the secret. Pods can consume secretsMaps as environment variables in a volume. You can specify the `data` field when creating a configuration file for a secret.
-
-To use a secret in an environment variable in a Pod, follow these steps.
-
-1. Create a secret or use an existing one. Multiple Pods can reference the same secret.
-2. Modify your Pod definition in each container, which you want to consume the value of a secret key, to add an environment variable for each secret key that you want to consume.
-3. Modify your image and/or command line so that the program looks for values in the specified environment variables.
-
-Pulsar clusters support using TLS or other authentication plugin for authentication.
+Function Mesh provides the `TLSSecret` and `AuthSecret` fields for Function, Source and Sink in the CRD definition. You can configure TLS encryption and/or TLS authentication using the following configurations.
```
Comment (teachwrite) [inline versioned_docs/version-0.1.6/connectors/io-crd-config/source-crd-config.md:57]: need a comma after "Source" in the first sentence ("...Function, Source, and Sink..")

File: versioned_docs/version-0.1.6/functions/function-crd.md [hunk 17]
```diff
@@ -83,32 +81,24 @@ When you specify a function or connector, you can optionally specify how much of
...
-
-To use a secret in an environment variable in a Pod, follow these steps.
-
-1. Create a secret or use an existing one. Multiple Pods can reference the same secret.
-2. Modify your Pod definition in each container, which you want to consume the value of a secret key, to add an environment variable for each secret key that you want to consume.
-3. Modify your image and/or command line so that the program looks for values in the specified environment variables.
+## Authentication
 
-Pulsar clusters supports using TLS or other authentication plugin for authentication.
+Function Mesh provides the `TLSSecret` and `AuthSecret` fields for Function, Source and Sink in the CRD definition. You can configure TLS encryption and/or TLS authentication using the following configurations.
```
Comment (teachwrite) [inline versioned_docs/version-0.1.6/functions/function-crd.md:86]: need a comma after "Source" in the first sentence ("...Function, Source, and Sink..")

File: versioned_docs/version-0.1.7/connectors/io-crd-config/sink-crd-config.md [hunk 18]
```diff
@@ -61,32 +61,24 @@ When you specify a function or connector, you can optionally specify how much of
...
-In Function Mesh, the secret is defined through a secretsMap. To use a secret, a Pod needs to reference the secret. Pods can consume secretsMaps as environment variables in a volume. You can specify the `data` field when creating a configuration file for a secret.
-
-To use a secret in an environment variable in a Pod, follow these steps.
-
-1. Create a secret or use an existing one. Multiple Pods can reference the same secret.
-2. Modify your Pod definition in each container, which you want to consume the value of a secret key, to add an environment variable for each secret key that you want to consume.
-3. Modify your image and or command line so that the program looks for values in the specified environment variables.
-
-Pulsar clusters support using TLS or other authentication plugin for authentication.
+Function Mesh provides the `TLSSecret` and `AuthSecret` fields for Function, Source and Sink in the CRD definition. You can configure TLS encryption and/or TLS authentication using the following configurations.
```
Comment (teachwrite) [inline versioned_docs/version-0.1.7/connectors/io-crd-config/sink-crd-config.md:66]: need a comma after "Source" in the first sentence ("...Function, Source, and Sink..")

File: versioned_docs/version-0.1.7/connectors/io-crd-config/source-crd-config.md [hunk 19]
```diff
@@ -54,32 +54,24 @@ When you specify a function or connector, you can optionally specify how much of
...
-In Function Mesh, the secret is defined through a secretsMap. To use a secret, a Pod needs to reference the secret. Pods can consume secretsMaps as environment variables in a volume. You can specify the `data` field when creating a configuration file for a secret.
-
-To use a secret in an environment variable in a Pod, follow these steps.
-
-1. Create a secret or use an existing one. Multiple Pods can reference the same secret.
-2. Modify your Pod definition in each container, which you want to consume the value of a secret key, to add an environment variable for each secret key that you want to consume.
-3. Modify your image and/or command line so that the program looks for values in the specified environment variables.
-
-Pulsar clusters support using TLS or other authentication plugin for authentication.
+Function Mesh provides the `TLSSecret` and `AuthSecret` fields for Function, Source and Sink in the CRD definition. You can configure TLS encryption and/or TLS authentication using the following configurations.
```
Comment (teachwrite) [inline versioned_docs/version-0.1.7/connectors/io-crd-config/source-crd-config.md:59]: need a comma after "Source" in the first sentence ("...Function, Source, and Sink..")

File: versioned_docs/version-0.1.7/functions/function-crd.md [hunk 20]
```diff
@@ -84,32 +82,24 @@ When you specify a function or connector, you can optionally specify how much of
...
-
-To use a secret in an environment variable in a Pod, follow these steps.
-
-1. Create a secret or use an existing one. Multiple Pods can reference the same secret.
-2. Modify your Pod definition in each container, which you want to consume the value of a secret key, to add an environment variable for each secret key that you want to consume.
-3. Modify your image and/or command line so that the program looks for values in the specified environment variables.
+## Authentication
 
-Pulsar clusters supports using TLS or other authentication plugin for authentication.
+Function Mesh provides the `TLSSecret` and `AuthSecret` fields for Function, Source and Sink in the CRD definition. You can configure TLS encryption and/or TLS authentication using the following configurations.
```
Comment (teachwrite) [inline versioned_docs/version-0.1.7/functions/function-crd.md:87]: need a comma after "Source" in the first sentence ("...Function, Source, and Sink..")

File: versioned_docs/version-0.1.7/scaling.md [hunk 21]
```diff
@@ -44,7 +44,7 @@ Function Mesh auto-scales the number of Pods based on the CPU usage, memory usag
 
 - Memory usage: auto-scale the number of Pods based on memory utilization.
   
-  This table lists built-in CPU-based autoscaling metrics. If these metrics cannot meet your requirements, you can auto-scale the number of Pods based on customized metrics defined in Pulsar Functions or connectors. For details, see [MetricSpec v2beta2 autoscaling](https://kubernetes.io/docs/reference/generated/kubernetes-api/v1.21/#metricspec-v2beta2-autoscaling).
+  This table lists built-in memory-based autoscaling metrics. If these metrics cannot meet your requirements, you can auto-scale the number of Pods based on customized metrics defined in Pulsar Functions or connectors. For details, see [MetricSpec v2beta2 autoscaling](https://kubernetes.io/docs/reference/generated/kubernetes-api/v1.21/#metricspec-v2beta2-autoscaling).
```
Comment (teachwrite) [inline versioned_docs/version-0.1.7/scaling.md:47]: "If these metrics do not meet..."

File: versioned_docs/version-0.1.8/connectors/io-crd-config/sink-crd-config.md [hunk 22]
```diff
@@ -61,32 +61,24 @@ When you specify a function or connector, you can optionally specify how much of
...
-In Function Mesh, the secret is defined through a secretsMap. To use a secret, a Pod needs to reference the secret. Pods can consume secretsMaps as environment variables in a volume. You can specify the `data` field when creating a configuration file for a secret.
-
-To use a secret in an environment variable in a Pod, follow these steps.
-
-1. Create a secret or use an existing one. Multiple Pods can reference the same secret.
-2. Modify your Pod definition in each container, which you want to consume the value of a secret key, to add an environment variable for each secret key that you want to consume.
-3. Modify your image and or command line so that the program looks for values in the specified environment variables.
-
-Pulsar clusters support using TLS or other authentication plugin for authentication.
+Function Mesh provides the `TLSSecret` and `AuthSecret` fields for Function, Source and Sink in the CRD definition. You can configure TLS encryption and/or TLS authentication using the following configurations.
```
Comment (teachwrite) [inline versioned_docs/version-0.1.8/connectors/io-crd-config/sink-crd-config.md:66]: need a comma after "Source" in the first sentence ("...Function, Source, and Sink..")

File: versioned_docs/version-0.1.8/connectors/io-crd-config/source-crd-config.md [hunk 23]
```diff
@@ -54,32 +54,24 @@ When you specify a function or connector, you can optionally specify how much of
...
-In Function Mesh, the secret is defined through a secretsMap. To use a secret, a Pod needs to reference the secret. Pods can consume secretsMaps as environment variables in a volume. You can specify the `data` field when creating a configuration file for a secret.
-
-To use a secret in an environment variable in a Pod, follow these steps.
-
-1. Create a secret or use an existing one. Multiple Pods can reference the same secret.
-2. Modify your Pod definition in each container, which you want to consume the value of a secret key, to add an environment variable for each secret key that you want to consume.
-3. Modify your image and/or command line so that the program looks for values in the specified environment variables.
-
-Pulsar clusters support using TLS or other authentication plugin for authentication.
+Function Mesh provides the `TLSSecret` and `AuthSecret` fields for Function, Source and Sink in the CRD definition. You can configure TLS encryption and/or TLS authentication using the following configurations.
```
Comment (teachwrite) [inline versioned_docs/version-0.1.8/connectors/io-crd-config/source-crd-config.md:59]: need a comma after "Source" in the first sentence ("...Function, Source, and Sink..")

File: versioned_docs/version-0.1.8/functions/function-crd.md [hunk 24]
```diff
@@ -84,32 +82,24 @@ When you specify a function or connector, you can optionally specify how much of
...
-
-To use a secret in an environment variable in a Pod, follow these steps.
-
-1. Create a secret or use an existing one. Multiple Pods can reference the same secret.
-2. Modify your Pod definition in each container, which you want to consume the value of a secret key, to add an environment variable for each secret key that you want to consume.
-3. Modify your image and/or command line so that the program looks for values in the specified environment variables.
+## Authentication
 
-Pulsar clusters supports using TLS or other authentication plugin for authentication.
+Function Mesh provides the `TLSSecret` and `AuthSecret` fields for Function, Source and Sink in the CRD definition. You can configure TLS encryption and/or TLS authentication using the following configurations.
```
Comment (teachwrite) [inline versioned_docs/version-0.1.8/functions/function-crd.md:87]: need a comma after "Source" in the first sentence ("...Function, Source, and Sink..")

File: versioned_docs/version-0.1.8/scaling.md [hunk 25]
```diff
@@ -44,7 +44,7 @@ Function Mesh auto-scales the number of Pods based on the CPU usage, memory usag
 
 - Memory usage: auto-scale the number of Pods based on memory utilization.
   
-  This table lists built-in CPU-based autoscaling metrics. If these metrics cannot meet your requirements, you can auto-scale the number of Pods based on customized metrics defined in Pulsar Functions or connectors. For details, see [MetricSpec v2beta2 autoscaling](https://kubernetes.io/docs/reference/generated/kubernetes-api/v1.21/#metricspec-v2beta2-autoscaling).
+  This table lists built-in memory-based autoscaling metrics. If these metrics cannot meet your requirements, you can auto-scale the number of Pods based on customized metrics defined in Pulsar Functions or connectors. For details, see [MetricSpec v2beta2 autoscaling](https://kubernetes.io/docs/reference/generated/kubernetes-api/v1.21/#metricspec-v2beta2-autoscaling).
```
Comment (teachwrite) [inline versioned_docs/version-0.1.8/scaling.md:47]: "If these metrics do not meet..."

**Review by teachwrite — CHANGES_REQUESTED** [review]
These are all small changes, and many are changes to the same sentence repeated through the documents. After you make these changes, you can go ahead and merge this PR.

**Comment by Huanli-Meng** [comment]
@teachwrite , I have applied your comments into docs. PTAL again. If you choose "Change request" when you submit your comments, i cannot merge the PR directly. Thanks.

**Pull request merged**